yarn-project/aztec-nr/aztec/src/oracle/get_nullifier_membership_witness.nr

```diff
@@ -1,50 +1,44 @@
 use dep::protocol_types::{
+    abis::nullifier_leaf_preimage::{
+        NullifierLeafPreimage,
+        NULLIFIER_LEAF_PREIMAGE_LENGTH,
+    },
     constants::NULLIFIER_TREE_HEIGHT,
     hash::pedersen_hash,
 };
 use crate::utils::arr_copy_slice;
 
-global LEAF_DATA_LENGTH: Field = 3;
-// TODO: move this to constants.hpp so that it gets computed as INDEX_LENGTH + LEAF_DATA_LENGTH + NULLIFIER_TREE_HEIGHT
+// INDEX_LENGTH + NULLIFIER_LEAF_PREIMAGE_LENGTH + NULLIFIER_TREE_HEIGHT
```

Comment:
Could be in the constants such that it is computed in there instead of being a custom value in here. The noir constants are used to generate the other values so should be fine for handling it?

Reply:
Is it now possible to compute a constant in Noir? `constants.hpp` is no longer used so as far as I know we don't have a way to do this anymore.

Reply:
Oh, I was thinking just moving it into the constants, but guess it is already based on stuff from constants so might be useless actually.